src/Microsoft.Deployment.DotNet.Releases/src/ReleaseVersion.cs

```diff
@@ -568,5 +559,281 @@ internal static int CompareNumericIdentifier(string a, string b)
 
             return r;
         }
+
+        /// <summary>
+        /// Converts the string representation of a version to an equivalent <see cref="ReleaseVersion"/> object.
+        /// </summary>
+        /// <param name="input">A string that contains a version to convert.</param>
+        /// <returns>An object that is equivalent to the version number specified in the <paramref name="input"/> parameter.</returns>
+        /// <exception cref="ArgumentNullException" />
```

Comment:
Why are these two exception elements empty?

Reply:
I didn't see the text show up in the IDE's intellisense tips so I wasn't sure if the text is ever used, but maybe it's just the IDE.